scripts/hello-world-deploy.sh

```diff
@@ -12,5 +12,10 @@ if ( which node )
 then
   echo "-> nodejs is installed"
 else
+  echo "-> Installing nodejs"
   yes | pacman -S nodejs
-fi
\ No newline at end of file
+fi
+
+npm install express
```

Comment:
Once you have a working project you will have a package.json file that contains all the node dependencies and you will just use `npm install` to get them all (or verify they are installed). 

This means that the commands to install expressjs aren't really part of the automation, you just them once to add it to the project and then rely on the package.json to keep track of all your installed dependencies.

So instead of `npm install express` here you would replace it with `npm install` which will install anything in the package.json.